src/actions/historyActions.js

```diff
@@ -217,9 +214,7 @@ export const fetchTransactionsHistoryAction = () => {
         relayerExtensionDevice?.address,
       );
 
-      const aaveHistory = await mapTransactionsHistoryWithAave(accountAddress, archanovaTransactionsHistory);
-      const poolTogetherHistory = await mapTransactionsPoolTogether(accountAddress, aaveHistory);
-      const sablierHistory = await mapTransactionsHistoryWithSablier(accountAddress, poolTogetherHistory);
+      const sablierHistory = await mapTransactionsHistoryWithSablier(accountAddress, archanovaTransactionsHistory);
       const rariHistory = await mapTransactionsHistoryWithRari(accountAddress, sablierHistory, ethereumSupportedAssets);
```

Comment:
I think we can gradually remove these mappings too.

Reply:
I recall that I removed all rari & sablier screens some time ago, maybe it's time to remove history entries as well?

Reply:
Yes, we still had UI mappings for Archanova history items, but I think the time has come for them too :)

Reply:
Will add to backlog.